src/layout/Layout.jsx

```diff
@@ -7,22 +7,35 @@ import i18n from '../locales/index.js'
 import JobDetails from '../components/JobDetails/JobDetails'
 import { fetchCatchmentJobs } from "../api/crosscutRequests"
 import { useToggle } from "../hooks/useToggle"
+import { setToken } from "../services/JWTManager";
 
-function Layout() {
+function Layout(props) {
     const [showCreateModal, setShowCreateModal] = useState(false)
     const [showInfoModal, setShowInfoModal] = useState(false)
     const [showJobDetailsModal, setShowJobDetailsModal] = useState(false)
     const [modalText, setModalText] = useState({ title: "", action: ""})
     const [jobs, setJobs] = useState()
     const [isToggled, toggle] = useToggle(false)
+    const { token } = props
+    setToken(token)
 
+    let poller 
     useEffect(() => {
-        fetchJobs()
+        poller = setInterval(() => {
+            fetchJobs()
+        }, 5000)
```

Comment:
Since this is a magic number, let's add a comment or a variable to define it (like "poll every 5s")